### PR TITLE
[CBRD-22856] Support Row Value Constructor

### PR DESCRIPTION
**Pull request by swi0110**
http://jira.cubrid.org/browse/CBRD-22856
Added to new TC for test to RVC.

Add list

- Row_Value_Constructor.sql
- Row_Value_Constructor_queryPlan.sql

### CODE REVIEW

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.queryPlan [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ssihil) [inline sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.queryPlan]: Is this file required?

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor.result [hunk 2]
```diff
@@ -0,0 +1,140 @@
+===================================================
```
Comment (ssihil) [inline sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor.result]: Is this file required?

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.queryPlan (hunk 1 above)
Comment (swi0110) [reply sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.queryPlan]: 'query plan' is empty contents file.
another test is the same this too.

(please refer to https://github.com/CUBRID/cubrid-testcases/blob/develop/sql/_14_mysql_compatibility_2/_15_host_variable/_12_common/cases/where_clause.queryPlan)

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor.result (hunk 2 above)
Comment (swi0110) [reply sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor.result]: sorry, this file is not needed.
I'm removed from this.

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+select /*+ recompile */ 1 from rvc_tmp2 where (col1,col2) in ((1,1)) and (col1,col2) in ((2,3),(2,2));
+select /*+ recompile */ * from rvc_tmp2 where (col1,col2,col3) in ((1,2,3),(1,1,1)) and col1 = 1 and col2 = 2 and col3 = 3;
+select /*+ recompile */ * from rvc_tmp2 where (nvl(col1,1),col2,col3) in ((1,1,1));
+select /*+ recompile */ * from rvc_tmp2 where (nvl(col1,1),col2,col3) in ((1,1,1),(2,2,2));
+select /*+ recompile */ * from rvc_tmp2 where (col2,nvl(col1,1),col4) in ((3,3,3),(2,1,4),(2,1,3));
+select /*+ recompile */ * from rvc_tmp2 where (col2,col1,lower(col4)) in ((3,3,3),(2,1,4),(2,1,3));
+select /*+ recompile  INDEX_SS */ * from rvc_tmp2 where col2 = 1 and col3 = 1;
+select /*+ recompile  INDEX_SS */ * from rvc_tmp2 where (col2,col3) = (1,1);
+select /*+ recompile  INDEX_SS */ * from rvc_tmp2 where (col2,col3) in ((1,1));
+select /*+ recompile  INDEX_SS */ * from rvc_tmp2 where (col2,col3) in ((1,1));
```
Comment (shparkcubrid) [inline sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql]: Why did you change the test case?
```
select /*+ recompile  INDEX_SS */ * from tmp where (col2,col3) in ((1,1),(2,3),(3,4));
```

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql [hunk 4]
```diff
@@ -0,0 +1,46 @@
+-- create table & idx
+create table rvc_tmp2(col1 int, col2 int, col3 int, col4 int, col5 int);
+insert into rvc_tmp2 values(1,2,3,4,5),(1,1,1,1,1),(2,2,2,2,2),(3,3,3,3,3);
+create index idx on rvc_tmp2(col1,col2,col3,col4);
+create index idx2 on rvc_tmp2(nvl(col1,1),col2,col3,col4);
+
+-- check for query plan
+set optimization level 513;
+
+select /*+ recompile */ * from rvc_tmp2 where (col1,col2) IN ((1,2),(2,2)) and col3 = 3 and col4 = 4;
```
Comment (shparkcubrid) [inline sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql]: How about adding Test case classification such as 'combination of predicates', 'combination of predicates with join'.

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql (hunk 3 above)
Comment (swi0110) [reply sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql]: It was my mistake.
I will modify that.

File: sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql (hunk 4 above)
Comment (swi0110) [reply sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql]: good idea! I will next refer to this.
